scylla/src/network/tls.rs

```diff
@@ -123,6 +123,7 @@ impl TlsConfig {
             TlsContext::OpenSsl010(ref context) => {
                 #[allow(unused_mut)]
                 let mut ssl = openssl::ssl::Ssl::new(context)?;
+                ssl.set_connect_state();
```

Comment:
> because it is clear what openssl method that we call when connecting that it is a tls client

I fail to parse this part of the commit message. Could you explain?